packages/webpack/src/runtime/debug.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @type {Record<string, { globals: Record<string, boolean> }>}
+ */
+const incrementalPolicy = {}
+
+const printPolicyDebug = () => {
+  const policy = JSON.stringify(incrementalPolicy, null, 2)
```

Comment:
does it matter that we're not using `json-stable-stringify` here?

Reply:
It's only here to make transferring the information collected into the policy-override.json - so it's a copy+paste+adjust and not copy-paste each global individually from console.